lib/ecto_morph.ex

```diff
@@ -47,7 +49,7 @@ defmodule EctoMorph do
         @primary_key nil
         embedded_schema do
           Enum.each(@properties, fn {key, %{"type" => type}} ->
```

Comment:
I'd pass in all the properties, or at least type and format:
https://json-schema.org/understanding-json-schema/reference/string.html#id7